src/common/types/metadata.js

```diff
@@ -55,7 +55,8 @@ type MetadataFieldValue = string | number | Array<any>;
 type MetadataFields = { [string]: MetadataFieldValue };
 
 type MetadataQueryInstanceTypeField = {
-    name: string,
+    displayName: string,
+    key: string,
```

Comment:
Leveraging metadata instance's `key` and `displayName` instead of custom `name` field.